### PR TITLE
Add PITS Globale Datenrettungsdienste as an adopter

### PR DESCRIPTION
**Pull request by arnoldberlin**
# Comprehensive Summary of your change
Added details in ADOPTER.md and uploaded logo. 

# Issue being fixed
Fixes #(issue)
N/A

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [ ] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by wy65701436** [comment]
@OrlinVasilev can you please help to review this?

**Comment by OrlinVasilev** [comment]
depends on https://github.com/goharbor/website/pull/477 !

**Review by OrlinVasilev — APPROVED** [review]
merge https://github.com/goharbor/website/pull/477 first

File: ADOPTERS.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ be added to this list as they transition to production deployments.
 
 <a href="https://www.jd.com" border="0" target="_blank"><img alt="JD.com" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/jd.png" height="50"></a>&nbsp; &nbsp; &nbsp;
 <a href="https://www.trendmicro.com" border="0" target="_blank"><img alt="trendmicro" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/trendmicro.png" height="50"></a>&nbsp; &nbsp; &nbsp; &nbsp;
+<a href="https://pitsdatenrettung.de/" target="_blank" border="0"><img alt="PITS Globale Datenrettungsdienste" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/pits-globale-datenrettungsdienste.svg" height="50"></a>&nbsp; &nbsp; &nbsp; &nbsp;
```
Comment (zyyw) [inline ADOPTERS.md:14]: Currently this dependent PR is merged, but the `src` URL(https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/pits-globale-datenrettungsdienste.svg) is still 404; Maybe the URL should be changed to:
https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/pits-globale-datenrettungsdienste.png

**Comment by zyyw** [comment]
@OrlinVasilev , do we have a process to validate that the identity of PR creator does actually represent this company.

**Comment by OrlinVasilev** [comment]
@zyyw - if we have concerns we can address that
but in that case I think it's not necessary
image

**Review by OrlinVasilev — APPROVED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**